astropy/io/fits/diff.py

```diff
@@ -262,6 +276,9 @@ def __init__(self, a, b, ignore_keywords=[], ignore_comments=[],
         ignore_blank_cards : bool, optional
             Ignore all cards that are blank, i.e. they only contain
             whitespace (default: True).
+
+        tolerance : float, optional
```

Comment:
I don't think that's possible because the parameter was replaced by 2 new parameters. I haven't considered such a case. 😅